buildtest/menu/__init__.py

```diff
@@ -88,10 +88,9 @@ def build_menu(self):
         ##################### buildtest build     ###########################
 
         parser_build.add_argument(
-            "-c",
-            "--config",
-            help="Specify test configuration file.",
-            metavar="TEST CONFIGURATION",
+            "-bs",
```

Comment:
Are you sure we want to have a “bs” flag?

Reply:
what about ``-b`` and ``--buildspec``

Reply:
I like it!

Reply:
This is resolved now in latest commit.